kpi/models/asset_version.py

```diff
@@ -54,14 +54,11 @@ def to_formpack_schema(self):
             'version_id_key': '__version__',
         }
 
-    def _content_hash(self):
+    @property
+    def content_hash(self):
         # used to determine changes in the content from version to version
-        # not saved, only compared with other asset_versions (in tests and
-        # migration scripts, initially)
-        _json_string = json.dumps({'version_content': self.version_content,
-                                   'deployed_content': self.deployed_content,
-                                   'deployed': self.deployed,
-                                   }, sort_keys=True)
+        # not saved, only compared with other asset_versions
+        _json_string = json.dumps(self.version_content,sort_keys=True)
```

Comment:
what about adding a lovely space after the comma something like `version_content,  sort_keys=true)`

Reply:
@jnm @noliveleger have you considered using https://www.pylint.org to not bother about such problems (plus avoid some coding errors)?

Reply:
@magicznyleszek, not a bad idea even if I already have a linter for python integrated in my IDE. 
Maybe when we upgrade to python3/django 2.x